tests/components/myuplink/conftest.py

```diff
@@ -64,34 +63,70 @@ async def setup_credentials(hass: HomeAssistant) -> None:
     )
 
 
+@pytest.fixture(scope="session")
+def device_fixture():
+    """Fixture for device."""
+    return Device(load_json_value_fixture("device.json", DOMAIN))
```

Comment:
Load the fixture into a string first in a separate fixture. That fixture should be scoped as session. Then use that fixture in this fixture that creates the device and is scoped per test.

That way we don't need to worry about modifying the device state in the tests, influencing other tests.

Reply:
OK